drupal/sites/default/boinc/themes/boinc/templates/forum-list.tpl.php

```diff
@@ -110,13 +110,13 @@
 
 <div class="fine-print">
   <p>
-    <?php print bts('Please be responsible in what you write and do not create posts which are offensive or insulting. Offensive posts or threads may be deleted by forum moderators without warning or discussion. Do not respond to offensive postings. Click on the "report" button at the bottom of the post to call it to the attention of the moderators.'); ?>
+    <?php print bts('Please be responsible in what you write and do not create posts which are offensive or insulting. Offensive posts or threads may be deleted by forum moderators without warning or discussion. Do not respond to offensive postings. Click on the "report" button at the bottom of the post to call it to the attention of the moderators.', array(), NULL, 'boinc:forum-fine-print'); ?>
   </p>
   <p>
     <?php print bts('We also ask that you keep all discussion on the message boards related to @project or BOINC with the small exception of the Science message board where you are free to discuss anything relevant to the underlying science. Participants interested in broader discussions should post to unofficial forums for @project.',
-      array('@project' => variable_get('site_name', bts('this project')))); ?>
+      array('@project' => variable_get('site_name', bts('this project'))), NULL, 'boinc:forum-fine-print'); ?>
```

Comment:
In this case, what we should do is find all instances of `variable_get('site_name', XYZ)`. Currently XYZ is very inconsistent. Sometimes its a string, sometimes its blank, sometimes it uses `bts()`, etc.

I propose we change XYZ to `'Drupal-BOINC'`, without `bts()` or `t()`. The idea being that as an admin you should **really** set the `site_name` variable. And that translators should not have to translate this default string that appears in case the admin does not `site_name`, because it should never show up.